### PR TITLE
Volunteer display niceties

### PR DESCRIPTION
**Pull request by idnorton**
* Add sha256 hashes for flask-admin modal dialogs
* Update volunteer base template to static_url_for

Both CSS and JS used links to external resources which causes issues for cross site scripting security measures. Other aspects of the site were updated but there's still a couple of hanger ons like this. See also https://github.com/emfcamp/Website/commit/b1b46e9b48374f25e05dd548e9a61119ed348a5e

* Populate volunteer email and name in test data
* Add display names for volunteer roles and venues

### CODE REVIEW

File: templates/volunteer/base.html [hunk 1]
```diff
@@ -5,8 +5,9 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>{% block title %}{% endblock %}{% if self.title() %} - {% endif %}EMF Volunteering</title>
-        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-        <link rel="stylesheet" href="{{ static_url_for('static', filename='css/volunteer.css') }}">
+        {% block css %}
+           <link rel="stylesheet" href="{{ static_url_for('static', filename="css/main.css") }}">
```
Comment (idnorton) [inline templates/volunteer/base.html]: This looks like `css/volunteer.css` was missed in translation, however the file no longer exists.

**Comment by idnorton** [comment]
I forgot a fixes #869

**Comment by idnorton** [comment]
UI changes from:
image

To:
image

**Comment by idnorton** [comment]
Edit UI changes from:
image

To:
image

**Comment by idnorton** [comment]
> Looks good to me, although this will break unless it's merged after #884 as this has the training `import`

Stray over from another branch, removed that as it shouldn't have been here.

**Comment by idnorton** [comment]
Fixed the black failures and also my lack of pre-commit.

**Comment by marksteward** [comment]
Looks good to me!

**Pull request merged**